### PR TITLE
open-cluster-management/backlog#6475 upd bm opt

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: manage_cluster/create_bare.adoc [hunk 1]
```diff
@@ -43,11 +42,32 @@ If the image that you want to use is not a standard image, you can enter the url
...
 . Select your hosts from the list of hosts that are associated with your provider connection.
 Select a minimum of three assets that are on the same bridge networks as the hypervisor.
-. *Optional:* Configure the cluster networking options.
+. Configure the cluster networking options.
++
+|===
+| Parameter | Description | Required or Optional
+
+| Base DNS domain | The base domain of your provider, which is used to create routes to your {ocp} cluster components. It is configured in your cluster provider's DNS as a Start of Authority (SOA) record. This setting cannot be changed after the cluster is created. | Required
+| Network type | The pod network provider plug-in to deploy. Only the OpenShiftSDN plug-in is supported on {ocp-short} 4.3. The OVNKubernetes plug-in is available as a Technology Preview on {ocp-short} 4.3. | Required
```
Comment (cdoan1) [inline manage_cluster/create_bare.adoc]: I need to look this up. We should list the default value as well.

Comment (cdoan1) [reply manage_cluster/create_bare.adoc]: regarding `Network type`

File: manage_cluster/create_bare.adoc [hunk 2]
```diff
@@ -43,11 +42,32 @@ If the image that you want to use is not a standard image, you can enter the url
...
+| Network host prefix | The subnet prefix length to assign to each individual node. For example, if hostPrefix is set to 23, then each node is assigned a /23 subnet out of the given CIDR, allowing for 510 (2^(32-23)-2) pod IP addresses. The default is 23. | Required
+| Service network CIDR | A block of IP addresses for services. OpenShiftSDN allows only one serviceNetwork block. The address must not overlap any other network block. The default value is 172.30.0.0/16. | Required
+| Machine CIDR | A block of IP addresses used by the {ocp-short} hosts. The address block must not overlap any other network block. The default value is 10.0.0.0/16. | Required
+| Provisioning network CIDR | The CIDR for the network to use for provisioning. The example format is: 172.30.0.0/16. | Required
+| Provisioning network interface | The name of the network interface on the control plane nodes that are connected to the provisioning network. | Optional
+| Provisioning network bridge | The name of the bridge on the hypervisor that is attached to the provisioning network. | Optional
```
Comment (cdoan1) [inline manage_cluster/create_bare.adoc]: These following 4 should not be `Optional`. In every bm deployment, we specify them.

File: manage_cluster/prov_conn_bare.adoc [hunk 3]
```diff
@@ -43,10 +43,29 @@ See https://access.redhat.com/documentation/en-us/openshift_container_platform/4
 . Add a list of your SSH known hosts.
 . For disconnected installations only: Complete the fields in the *Configuration for disconnected installation* subsection with the required information:
 +
-* _Image Registry Mirror_: This optional value contains the disconnected registry path. The path contains the hostname, port, and repository path to all of the installation images for disconnected installations. Example: `repository.com:5000/openshift/ocp-release`.
+* _Image Registry Mirror_: This value contains the disconnected registry path. The path contains the hostname, port, and repository path to all of the installation images for disconnected installations. Example: `repository.com:5000/openshift/ocp-release`.
```
Comment (cdoan1) [inline manage_cluster/prov_conn_bare.adoc:46]: The path will create an image content source policy mapping in the `install-config.yaml` to the OCP release images. Example: `repository.com:5000` would produce this imageContentSource stanza:

```
imageContentSources:
- mirrors:
  - registry.example.com:5000/ocp4
  source: quay.io/openshift-release-dev/ocp-release-nightly
- mirrors:
  - registry.example.com:5000/ocp4
  source: quay.io/openshift-release-dev/ocp-release
- mirrors:
  - registry.example.com:5000/ocp4
  source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
```
(if i added any amazonaws url example, lets make sure to replace with generic uri.)

**Review by cdoan1 — APPROVED** [review]
Please see comments. I approve ahead of changes, just ping me if anything is unclear!

File: manage_cluster/create_bare.adoc (hunk 1 above)
Comment (cadawson) [reply manage_cluster/create_bare.adoc]: It looks like OVNKubernetes was the default value in the instance that I saw. I added this, but let me know if that is not how it is now.

File: manage_cluster/create_bare.adoc (hunk 2 above)
Comment (cadawson) [reply manage_cluster/create_bare.adoc]: These are now indicated as Required. The Ingress VIP remained Optional.

File: manage_cluster/prov_conn_bare.adoc (hunk 3 above)
Comment (cadawson) [reply manage_cluster/prov_conn_bare.adoc:46]: Replaced the note and existing example with this example.

**Pull request merged**